src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx

```diff
@@ -0,0 +1,263 @@
+                        ],
+                    },
+                    {
+                        localIdentifier: "view",
+                        items: [
+                            {
+                                visualizationAttribute: {
+                                    localIdentifier: "viewId",
+                                    displayForm: { uri: "/gdc/md/projectId/obj/1024" },
+                                },
+                            },
+                        ],
+                    },
+                ],
+                visualizationClass: { uri: "/gdc/md/projectId/obj/75535" },
+            };
+
+            const wrapper = createComponent({
+                ...defaultProps,
+                mdObject,
+            });
+
+            const axisSections = wrapper.find(NameSubsection);
+
+            const xAxisSection = axisSections.at(0);
+            expect(xAxisSection.props().disabled).toEqual(false);
+
+            const yAxisSection = axisSections.at(1);
+            expect(yAxisSection.props().disabled).toEqual(false);
+        });
+
+        it("should render configuration panel with enabled name sections in dual axis chart", () => {
+            const mdObject = {
+                buckets: [
+                    {
+                        localIdentifier: "measures",
+                        items: [
+                            {
+                                measure: {
+                                    localIdentifier: "measureId",
+                                    definition: {
+                                        measureDefinition: {
+                                            item: { uri: "/gdc/md/projectId/obj/9211" },
+                                        },
+                                    },
+                                },
+                            },
+                            {
+                                measure: {
+                                    localIdentifier: "measureId2",
+                                    definition: {
+                                        measureDefinition: {
+                                            item: { uri: "/gdc/md/projectId/obj/9203" },
+                                        },
+                                    },
+                                },
+                            },
+                        ],
+                    },
+                    {
+                        localIdentifier: "view",
+                        items: [
+                            {
+                                visualizationAttribute: {
+                                    localIdentifier: "viewId",
+                                    displayForm: { uri: "/gdc/md/projectId/obj/1024" },
+                                },
+                            },
+                        ],
+                    },
+                ],
+                visualizationClass: { uri: "/gdc/md/projectId/obj/75535" },
+            };
```

Comment:
Duplicate bucket items. Should declare some variable and reuse them.

Reply:
No, I don't think it's really neccessary.

Reply:
you mean that leave duplication here is better.
it isn't look good to me.